### ISSUE TITLE
MVC Getting started: Model: Update scaffolding command and table to --databaseProvider sqlite

### ISSUE THREAD

**Issue by wadepickett** [dotnet/AspNetCore.Docs#28815]
The scaffolding command has changed and needs to be updated in this topic. 

--useSqlite | -sqlite is now obsolete.   

Scaffolding now provides this instruction, prompting this change:
`  --databaseProvider|-dbProvider : Database provider to use. Options include 'sqlserver' (default), 'sqlite', 'cosmos', '`

The scaffolding command for the VSC and VS for Mac tabs needs to be updated to this:
```dotnetcli
dotnet aspnet-codegenerator controller -name MoviesController -m Movie -dc MvcMovie.Data.MvcMovieContext --relativeFolderPath Controllers --useDefaultLayout --referenceScriptLibraries --databaseProvider sqlite
```

Also update the table following the command sample to reflect that change to something like: 
--databaseProvider sqlite | Flag to specify if DbContext should use SQLite instead of SQL Server.

For the VS for Mac version we may want to consider removing the screenshot since it will always show a package version.  The packages are updated quickly enough that the screenshot can imply an earlier version rather than the latest stable version is that package to look for.

Related to Razor Pages tutorial issue #29052.  A little different since the Razor Pages version Uses the Scafolding UI for VS for Mac rather than CLI.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 1d975a83-1550-b284-d0d8-7fe2ea206ff5
* Version Independent ID: 48e81b42-7c2b-afbf-71e6-d4f442a4bd1f
* Content: [Part 4, add a model to an ASP.NET Core MVC app](https://learn.microsoft.com/en-us/aspnet/core/tutorials/first-mvc-app/adding-model?view=aspnetcore-8.0&tabs=visual-studio)
* Content Source: [aspnetcore/tutorials/first-mvc-app/adding-model.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/tutorials/first-mvc-app/adding-model.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-tutorials**
* GitHub Login: @wadepickett
* Microsoft Alias: **wpickett**

---
[Associated WorkItem - 80806](https://dev.azure.com/msft-skilling/Content/_workitems/edit/80806)

**Comment by wadepickett** [comment]
I should address this issue while updating to 8.0 for this series in issue #28669

**Comment by wadepickett** [comment]
Fixed with #29115
